### DISCUSSION TITLE
Reddit Ads data export

### DISCUSSION THREAD

**Discussion by rubanvo** [OWOX/owox-data-marts#2] [Ideas]
Hi!

I'm looking for a solution to export data about costs, clicks and impressions from a Reddit Ads account to Google Sheets. Do you have any ideas or advice for this?

Thanks in advance!

**Comment by prokopetsv** [comment]
Hello! Thank you for starting the discussion. Please [refer to the documentation for the implemented Reddit Ads connector](https://github.com/OWOX/owox-data-marts/blob/main/packages/connectors/src/Sources/RedditAds/GETTING_STARTED.md). You can upload data to BigQuery and Google Sheets using the connector-based data mart. Should you have any questions during the process, please let us know.
